src/components/action-popover/action-popover.test.tsx

```diff
@@ -448,6 +447,7 @@ test("clicking the menu button focuses the first focusable element", async () =>
   );
 
   await user.click(screen.getByRole("button"));
+  jest.runAllTimers();
```

Comment:
comment: might be worth using pending timers here as I don't think we need to run both macro and micro tasks